src/styles/utils/reset.scss

```diff
@@ -0,0 +1,9 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
it would be more correct to specify elemnt for which the margin and padding reset will need to be used. For example:
```
h1, h2, p {
  margin: 0;
  padding: 0;
}
```